catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/utils/BrickTestUtils.java

```diff
@@ -84,4 +98,36 @@ public static void createUserListFromDataFragment(String userListName, boolean f
 		onView(withText(userListName))
 				.check(matches(isDisplayed()));
 	}
+
+	public static void addBrick(Class<?> brickHeaderClass, int brickCategoryId) {
```

Comment:
as far as i can see this function isnt called anywhere in this PR.
is this necessary?